lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -112,6 +112,13 @@ defmodule LivebookWeb.SessionLive.CellComponent do
             <%= remix_icon("arrow-up-s-line", class: "text-xl") %>
           </button>
         </span>
+        <span class="tooltip top" aria-label="Capture code snippet">
+          <button class="icon-button"
+            data-element="capture-code-snippet"
+            phx-value-cell_id="<%= @cell_view.id %>">
+            <%= remix_icon("camera-2-line", class: "text-xl") %>
```

Comment:
Placing the button between up and down icons is not intuitive, let's move this after the link icon.

Reply:
Yeah, noticed this too. Moved it to the right of cell settings.

Reply:
Moved it now according to your suggestion.